jest.config.ts

```diff
@@ -1,5 +1,5 @@
-import { getJestProjects } from '@nx/jest';
+import { getJestProjectsAsync } from '@nx/jest';
 
-export default {
-  projects: getJestProjects(),
-};
+export default async () => ({
+  projects: await getJestProjectsAsync(),
```

Comment:
Do you know why this chance was necessary? It doesn't really matter, I'm just curious.

Reply:
It was automated. I saw it too and thought it was interesting.

I would guess this just blocked before, and now because it's async it's note and that probably allows the node process to do whatever other things while it waits for this, hypothetically improving speed.